beacon/engine/types.go

```diff
@@ -60,6 +60,8 @@ type PayloadAttributes struct {
 	// and contains encoded EIP-1559 parameters. See:
 	// https://github.com/ethereum-optimism/specs/blob/main/specs/protocol/holocene/exec-engine.md#eip1559params-encoding
 	EIP1559Params []byte `json:"eip1559Params,omitempty" gencodec:"optional"`
+	// MinBaseFee is a field for rollups implementing the minimum base fee feature.
+	MinBaseFee uint64 `json:"minBaseFee,omitempty" gencodec:"optional"`
```

Comment:
Spec: https://specs.optimism.io/protocol/jovian/exec-engine.html#minimum-base-fee-in-payloadattributesv3

Reply:
If we go this way, the specs should be updated to be explicit about the requirement to be `nil` pre Jovian.

Reply:
I agree.